src/app-layout/drawer/styles.scss

```diff
@@ -63,34 +62,26 @@ $drawer-z-index-mobile: 1001;
   }
 }
 
-.trigger {
-  .drawer-content > .drawer-triggers > & {
```

Comment:
Before this PR, app layout used our button component and this line overrode button styles. 

Overriding styles is bad even if done for a good reason, it still may cause issues. 

In this PR I replaced it with custom styles on top of native `<button>` tag (see the `<ToggleButton>` component) which has the correct styles from the start, without overrides